modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro

```diff
@@ -984,6 +985,12 @@ definition {
 						locator1 = "ObjectAction#INPUT_NEW_VALUE",
 						value1 = ${inputExpressionValue});
 				}
+
+				if (IsElementPresent(locator1 = "ObjectAction#ADD_ERROR_MESSAGE_VALIDATION")) {
+					Type(
+						locator1 = "ObjectAction#ADD_ERROR_MESSAGE_VALIDATION",
+						value1 = ${errorMessage});
+				}
```

Comment:
I'm seeing this change in another test in https://github.com/liferay-bpm-qa/liferay-portal/pull/451. If we need this change across multiple tests, we should coordinate this to avoid merge conflicts.

A good option would be to cherry pick the change from an existing branch and add it to your own.

Reply:
Right @timpak, I appreciate for feedback, I'll make the changes.